src/index.d.ts

```diff
@@ -16,7 +16,7 @@ export type StandardProps<C, ClassKey extends string, Removals extends keyof C =
     style?: Partial<React.CSSProperties>;
   };
 
-export type Contrast = 'light' | 'dark';
+export type Type = 'light' | 'dark';
```

Comment:
`Type` is an extremely generic name, especially to be exporting from the root module. Maybe call this `ColorDirection` to match the terminology in `createLightOrDark` function?

Reply:
Mmph. I used "direction" as a placeholder name, and then couldn't come up with anything better, but figured it didn't matter so much as it's internal, and its subsequent use hints as to its purpose. Also, light and dark shades are very different in purpose to light and dark themes.

I can't come up with anything better though... 😖

Reply:
Yeah, I'd say `Brightness` or `ColorType` are better names for the type.

Reply:
I'm going with `PaletteType`.